universal-application-tool-0.0.1/conf/routes

```diff
@@ -15,7 +15,9 @@ POST    /postgres                   controllers.PostgresController.create(reques
 POST    /postgresSync               controllers.PostgresController.createSync(request: Request)
 
 # A controller for a page for an admin to create a new application
-GET     /admin/create               controllers.admin.CreateAppController.index
+GET     /admin/programs             controllers.admin.AdminProgramController.list
```

Comment:
issue (blocking): Our [Html routing conventions](https://github.com/seattle-uat/universal-application-tool#html-routing-convention) uses `index` for the controller method name for listing a resource. We should either use that here, or if there's a good reason to use `list` instead, update the README.

Reply:
done!